cmd/routed-eni-cni-plugin/cni.go

```diff
@@ -237,9 +237,11 @@ func add(args *skel.CmdArgs, cniTypes typeswrapper.CNITYPES, grpcClient grpcwrap
 }
 
 // generateHostVethName returns a name to be used on the host-side veth device.
-func generateHostVethName(prefix, netname, containerid, ifname string) string {
+// The veth name is generated such that it aligns with the value expected
+// by Calico for NetworkPolicy enforcement.
```

Comment:
Is there a standard being followed in this naming convention? If not, we should treat this a stop-gap solution with a TODO followup as Calico or any other plugins (that might integrate in similar fashion) may break without a warning

Reply:
Definitely agree. This kind of hidden coupling is something we should really try to avoid.

Apparently kind has to do something similar, see https://github.com/kubernetes-sigs/kind/issues/842 and the code in  https://github.com/cybozu/neco-containers/blob/cddc3696f72fbc80580d088f338ef42d8f4ae7cf/kind-node/src/ptp/ptp.go#L414